apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.html.ts

```diff
@@ -57,6 +68,21 @@ export function AdministratorInstitutions({
                   ${institutions.map(
                     ({ institution, authn_providers }) => html`
                       <tr>
+                        <td>
+                          <button
+                            type="button"
+                            class="btn btn-sm"
+                            data-toggle="modal"
+                            data-target="#edit-institution-modal-${institution.id}"
+                          >
+                            <i class="fas fa-edit"></i>
+                          </button>
+                          ${editInstitutionModal({
+                            csrf_token: resLocals.__csrf_token,
+                            institution,
+                            authn_providers,
+                          })}
+                        </td>
```

Comment:
Rather than editing inline on this page, I'd prefer to move the edit form into the institution settings page (that is, what you get when you click on the institution name, something like `/pl/institution/1/admin`. This page is implemented at `apps/prairielearn/src/ee/pages/institutionAdminGeneral`.

Reply:
I have moved this form into `/ee/pages/institutionAdminGeneral` in commit [402fad6](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/402fad6c5a45a0e8ff5187613d3563a1bbf7d8f0). However, this route seemed inaccessible so I added a new route in `server.js` to reach this page at `pl/institution/:instition_id/admin`. I raised a question about this in my code review as well, but let me know if I am using this incorrectly.

Reply:
You'll have to update your config file to include `"isEnterprise": true`, which our [EE license](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/ee/LICENSE) specifically allows for development and testing purposes.

Reply:
This cleared everything up! Sorry, didn't realize I needed to update to `isEnterprise: true` locally. The requested changes now make so much sense.